manifests/g/Google/GCPW/70.103.49152/Google.GCPW.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Google.GCPW
+PackageVersion: 70.103.49152
+InstallerLocale: en-US
+InstallerType: wix
+ProductCode: '{29716494-E5B9-3E21-9524-3160129C8781}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://dl.google.com/credentialprovider/gcpwstandaloneenterprise64.msi
+  InstallerSha256: 96C2877E316D1D7D1A4F90EBCAA638B6632BD3FE6F63A89E632F2E2798377CDC

```

```suggestion
- Architecture: x64
  InstallerUrl: https://dl.google.com/credentialprovider/gcpwstandaloneenterprise64.msi
  InstallerSha256: 96C2877E316D1D7D1A4F90EBCAA638B6632BD3FE6F63A89E632F2E2798377CDC
- Architecture: x86
  InstallerUrl: https://dl.google.com/credentialprovider/gcpwstandaloneenterprise.msi
  InstallerSha256: 2864C81BDD73A4E8EB094133078501F039E5A2F3DAD2BCA19102FE1F8DCB6D22
```
Both architectures should be included

---

Regarding this, I wanted to ask if there is a way to add/update a package with another architecture using `wingetcreate`, because I could not find a way to do it.